src/vpk/Velopack.Vpk/Commands/Flow/PublishCommand.cs

```diff
@@ -23,5 +25,10 @@ public PublishCommand()
 
         AddOption<bool>(v => WaitForLive = v, "--waitForLive")
             .SetDescription("Wait for the release to finish processing and go live.");
+
+        AddOption<int>(v => TieredRolloutPercentage = v, "--tieredRolloutPercentage")
```

Comment:
this is a bit wordy. I don't mind if you feel it's right, but `--rolloutPercent` or `--initialRollout` might be a bit shorter?

Reply:
I like those names better